* TOC
{:toc}

# Recent PhD Graduate

#### Technical Skills: Python, R, LaTex

## Profile Summary

PhD in Molecular Pathology with expertise in data-driven research and advanced analytical techniques, especially in virology and immunology. Conducted RNAseq data analysis to identify therapeutic targets and managed complex research projects from start to finish. Experienced in presenting scientific findings at international conferences and through public outreach. Seeking to leverage research experience and strategic thinking in a consulting role within the biopharmaceutical industry.

## Education

### PhD, Molecular Virology @ The University of Cambridge, England (_July 2024_)

**Thesis:** ["Analysis of Human and Bovine IFIT Protein Interaction with Respiratory Syncytial Virus"](https://doi.org/10.17863/CAM.110920)

- **Key Areas:** Virology, Immunology, Protein Interactions, Vaccine Development, Therapeutics, Genetic Sequencing
- **Relevant Coursework:** Data Science in Python, Intermediate Supervised Machine Learning, Core Statistics with R
- Developed a strong expertise in translational research, applying advanced machine learning techniques to analyze large datasets (RNAseq) and uncover critical insights into viral mechanisms and protein interactions. Mastered RT-qPCR, Western Blot, Confocal Imaging, Immunoprecipitation, and viral infection assays.

### MSci, Medical Sciences (Pharmacology) @ The University of Aberdeen, Scotland (_July 2019_)

**Classification:** First-class honours (4 GPA equivalent)

- **Key Areas:** Pharmacology, Physiology, Pathology, Healthcare, Therapeutics, Clinical Trial Design
- Gained in-depth knowledge of healthcare systems, drug development, and clinical research methodologies. Solidified a foundation in data-driven approaches to pharmacological research, focusing on therapeutics.

## Work Experience

### PhD Candidate @ [University of Cambridge](https://www.path.cam.ac.uk/directory/ian-brierley) (_October 2019 - July 2024_)

-	**Data-Driven Research:** Developed and implemented machine learning models to analyze RNAseq data from over 150 samples, improving predictive accuracy by 20% and identifying novel therapeutic targets.
-	**Advanced Experimental Techniques:** Utilized RT-qPCR, Western Blot, Confocal Imaging, RNAseq, and Immunoprecipitation to validate findings and advance the field of viral immunology.
-	**Project Management:** Led a multi-year research project, managing timelines, resources, and collaborating across multidisciplinary teams.
-	**Outreach:** Successfully communicated complex data to diverse audiences at international conferences, contributing to the scientific community and enhancing collaborative efforts. Engaged in science communication by visiting schools to explain scientific concepts, fostering a passion for science in younger audiences.

Skills: RT-qPCR, Western Blot, Confocal Imaging, RNAseq, Immunoprecipitation, PCR

### Undergraduate Research Fellow @ [University of Aberdeen](https://www.abdn.ac.uk/people/c.debari#research) (_January 2019 - March 2019_)

**Project:** [Developing RNA interference technologies for targeting stem cells in joint tissues](https://drive.google.com/file/d/1k2Eq1qXjQn7hznZHdQe3GSyw74BeMmlE/view?usp=sharing)
-	Conducted statistical analysis and data visualisation of histological samples.
-	Collaborated with a team of researchers to develop and optimise semi-automatic experimental protocols, resulting in a 47% reduction in analysis time while also reducing bias, resulting in more accurate outcomes.

Skills: RT-qPCR, tissue staining, QuPath

### Undergraduate Research Fellow @ [EMBL Heidelberg](https://www.embl.org/groups/gavin/) (_July 2017 - July 2018_)

**Project:** [Characterisation of lipid transfer proteins](https://drive.google.com/file/d/1cywV5b0tLoMWPHP6JNHy7ENHYjmXwQ7E/view?usp=sharing)
-	Prepared and analysed plasmid libraries for 106 EGFP-tagged lipid transfer proteins, streamlining the workflow and reducing preparation time by 15%.

Skills: agarose gel DNA electrophoresis, molecular cloning

## Projects

### Data Analytics

#### Project 1: [Analysis of Forecasting Competition Aggregation Method Performance](https://github.com/MisokralPanovic/Forecasting-Competition-Data-Analysis)

This report investigates the performance of different aggregation methods for forecasting competition assessment, using the RCT-A dataset from the HFC competition. I evaluated the five aggregation methods and proposed an improvement based on the best-performing method.

The dataset was analysed using the data.table R package, which allows fast and memory efficient handling of data.

**Skills:** R data.tables, RMarkdown

#### Project 2: [Web Scraping Tabular Data with Python](https://github.com/MisokralPanovic/Web-Scraping-Tabular-Data)

In this project, I performed web scraping on the Wikipedia page for “List of Old-Growth Forests” to collect, clean, and analyze forest data from various regions worldwide. Using Python’s BeautifulSoup library, I extracted data from multiple tables with inconsistent formatting and missing information, employing custom functions to standardize the dataset for analysis. I used techniques for handling data inconsistencies, structuring a custom scraping pipeline, and conducting exploratory data analysis to answer questions about forest distribution by country and region.

**Skills:** Python, BeautifulSoup, web scraping, data cleaning, regex, exploratory data analysis
<!--
#### Project 4: [Some stuff with Tableau](link to github page)
Description text of what it is

Picture from analysis

**Skills:** tableau

### Data Science

#### Project 1: [OpenCV ML Confocal Image Analysis](link to github page)

Description text of what it is

Picture from analysis

**Skills:**

#### Project 2: [Name of project](link to github page)

Description text of what it is

Picture from analysis

**Skills:**

#### Project 3: [Name of project](link to github page)

Description text of what it is

Picture from analysis

**Skills:**
-->
### Life Science

#### Project 1: [qPCR Analysis Pipeline](https://github.com/MisokralPanovic/qPCR-Analysis-Template)

I developed a robust qPCR analysis pipeline in R to address the challenges of varying amplification efficiencies across different primers. The pipeline constructs a standard curve for accurate copy number extrapolation, normalizes target gene data using log2 deltaCt values of a housekeeping gene, and outputs adjusted copy numbers for reliable comparison across experimental conditions. Using R and the tidyverse, I automated data manipulation, statistical analysis, and visualization within an RMarkdown document, producing a reproducible and transparent workflow for quantitative gene expression analysis in life sciences.

![](https://raw.githubusercontent.com/MisokralPanovic/qPCR-Analysis-Template/refs/heads/main/Figures/plot_normalised_values_bIFIT1.png)

**Skills:** R (tidyverse), Data Wrangling and Transformation, Statistical Analysis, Data Visualization (ggplot2), Scientific Data Interpretation, RMarkdown Automation and Reproducibility

#### Project 2: [RNAseq Pipeline](https://github.com/MisokralPanovic/RNAseq)

I conducted a differential expression analysis on RNA-seq data to identify genes with significant expression changes between treatment conditions. Using the DESeq2 package, I applied techniques such as normalization, dispersion estimation, and variance stabilization to process the raw counts, ensuring robust statistical testing. Exploratory analyses included principal component analysis (PCA) and sample distance clustering, which validated sample groupings and data quality. I further enhanced the results by adding gene annotations, preparing the data for downstream gene ontology (GO) enrichment analyses.

![](https://raw.githubusercontent.com/MisokralPanovic/RNAseq/refs/heads/main/report/RNAseq-Pipeline-Report_files/figure-gfm/top_var_heatmap-1.png) 

#### Project 3: [PhD Thesis](https://github.com/MisokralPanovic/Thesis)

This repository contains the LaTeX source files and supporting materials for my PhD thesis, which was successfully defended at the University of Cambridge. The project demonstrates my ability to:

- **Write in LaTeX:** I used LaTeX for the entire document creation process, showcasing my proficiency with the platform, particularly for managing complex academic documents.
- **Work with Large Documents:** The thesis, comprising multiple chapters, appendices, figures, tables, and references, highlights my ability to structure and organize extensive documents efficiently.
- **Academic/Medical Writing:** The content focuses on advanced topics in molecular pathology, pharmacology, and virology, evidencing my ability to communicate complex scientific research clearly and effectively.
- **File and Project Management:** This repository illustrates my ability to manage a large-scale writing project, including version control, organizing LaTeX files, and handling the compilation of a document with multiple components.

**Skills:** LaTex, Git, GitHub, academic writing, file management
<!--
### Miscellaneous

#### Project 1: [Web Application Development with Django](link to github page)
Description text of what it is

Picture from analysis

**Skills:** django
-->
## Skills

### Data Analytics Skills

- **Programming:** python (pandas, numpy, matplotlib), R (tidyverse, data.table), Markdown
- **Tools & Platforms:** Git, Jupyter Notebooks, Hadoop Ecosystem, UNIX/Bash
- **Subject Knowledge:** Hypothesis Testing, Data Mining, Statistical Analysis, Advanced Data Visualization (Excel, Power BI, Tableau)
- **Theoretical knowledge** of big data, text mining, predictive modelling, SQL, and agile management methodology

### Data Science Skills

- **Programming:** python (tensorflow), R (caret), Weka, Snowflake, Markdown
<!-- - **Tools & Platforms:** -->
- **Subject Knowledge:** Predictive Modelling,

### Consultancy Skills

- **Programming:** Excel, LaTex
- **Tools & Platforms:** PowerPoint, Word

### Life Science Skills

- **Programming:** SAS, JASP, Jamovi, R (DESeq2), python (biopython), QuPath, CellProfiler, PyMol
<!-- - **Tools & Platforms:**  Websites and tools like stuff for genealogy tree, genome annotation, mapping, etc (check the master table) -->
- **Lab Technical Expertise:** RT-qPCR, Western Blot, Confocal Imaging, RNAseq, Immunoprecipitation, Cell Culture, Viral Infection Assays
- **Healthcare & Research:** Extensive experience in molecular pathology and therapeutic research, with a focus on translating complex data into actionable insights for drug and vaccine development.
<!-- - **Subject Knowledge:**  -->

### General

- **Project Management and Leadership:** Resource Coordination, Experiment Planning (Gantt Charts, Weekly Plans), Mentoring Junior Researchers
- **Languages:** Slovak (Native), English (Bilingual)

## Conferences and Talks

| Event | Place | Year | Action |
| --- | --- | --- | --- |
| MicroSoc Conference | Belfast | 2022 | Poster Presentation |
| Internal Seminar Series | University of Cambridge | 2023 | Talk |


## Courses
<!--
### Data Analytics Courses

<table>
  <thead>
    <tr>
      <th><strong>Technology</strong></th>
      <th><strong>Course Name</strong></th>
      <th><strong>Provider</strong></th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td></td>
      <td></td>
    </tr>
  </tbody>
</table>

### Data Science Courses

<table>
  <thead>
    <tr>
      <th><strong>Technology</strong></th>
      <th><strong>Course Name</strong></th>
      <th><strong>Provider</strong></th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>caret</td>
      <td>The caret R package</td>
      <td>University of Cambridge</td>
    </tr>
  </tbody>
</table>

### Consultancy Courses

<table>
  <thead>
    <tr>
      <th><strong>Technology</strong></th>
      <th><strong>Course Name</strong></th>
      <th><strong>Provider</strong></th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td></td>
      <td></td>
    </tr>
  </tbody>
</table>
-->
### Life Science Courses

<table>
  <thead>
    <tr>
      <th><strong>Technology</strong></th>
      <th><strong>Course Name</strong></th>
      <th><strong>Provider</strong></th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>RNAseq</td>
      <td>Transcriptome Analysis for Non-Model Organisms</td>
      <td>University of Cambridge</td>
    </tr>
    <tr>
      <td>Ensembl REST API</td>
      <td>Ensembl REST API Course</td>
      <td>University of Cambridge</td>
    </tr>
    <tr>
      <td>Ensembl Browser</td>
      <td>Ensembl Browser Workshop</td>
      <td>University of Cambridge</td>
    </tr>
    <tr>
      <td>ArrayExpress,  Expression Atlas</td>
      <td>Transcriptomics Data and Tools</td>
      <td>EMBL EBI</td>
    </tr>
  </tbody>
</table>

## Publications

1. **Varga, M.** (2024). The Interplay of RSV and IFITs: A comprehensive analysis of human and bovine Interferon-Induced Proteins with Tetratricopeptide Repeats induction and interaction with human and bovine Respiratory Syncytial Virus [Apollo - University of Cambridge Repository]. [https://doi.org/10.17863/CAM.110920](https://doi.org/10.17863/CAM.110920)
2. Conceicao, C., Thakur, N., Human, S., Kelly, J. T., Logan, L., Bialy, D., Bhat, S., Stevenson-Leggett, P., Zagrajek, A. K., Hollinghurst, P., **Varga, M.**, Tsirigoti, C., Tully, M., Chiu, C., Moffat, K., Silesian, A. P., Hammond, J. A., Maier, H. J., Bickerton, E., Shelton, H., Dietrich, I., Graham, S. C., Bailey, D. (2020). The SARS-CoV-2 Spike protein has a broad tropism for mammalian ACE2 proteins. PLOS Biology, 18(12), e3001016. [https://doi.org/10.1371/journal.pbio.3001016](https://doi.org/10.1371/journal.pbio.3001016)
